Source/Core/Common/VR/DolphinVR.cpp

```diff
@@ -0,0 +1,175 @@
+// Copyright 2016 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include <cstring>
+
+#include "Common/VR/DolphinVR.h"
+#include "Common/VR/VRBase.h"
+#include "Common/VR/VRInput.h"
+#include "Common/VR/VRMath.h"
+#include "Common/VR/VRRenderer.h"
+
+namespace Common::VR
+{
+Base* s_module_base = NULL;
+Input* s_module_input = NULL;
+Renderer* s_module_renderer = NULL;
+bool s_platform_flags[PLATFORM_MAX];
+
+static void (*UpdateInput)(int id, int l, int r, float x, float y, float jx, float jy);
+
+/*
+================================================================================
+
+VR app flow integration
+
+================================================================================
+*/
+
+bool IsEnabled()
```

Comment:
When will `OPENXR` be true for official builds?  Do we plan on having another target?  @JosJuice  has this been discussed with the team how this will be distributed?

Reply:
Someone would have to implement a PCVR version (most of the code could be reused). I work only on a standalone Android-based VR headset version.

Reply:
> has this been discussed with the team how this will be distributed?

No, nothing yet.